lib/core/src/Cardano/Wallet/Primitive/Types/UTxO/Gen.hs

```diff
@@ -66,3 +69,15 @@ genEntryLargeRange = (,)
     -- Note that we don't need to choose outputs from a large range, as inputs
     -- are already chosen from a large range:
     <*> genTxOut
+
+--------------------------------------------------------------------------------
+-- Selecting random UTxO entries
+--------------------------------------------------------------------------------
+
+-- | Selects up to a given number of entries at random from the given UTxO set.
+--
+-- Returns the selected entries and the remaining UTxO set with the entries
+-- removed.
+--
+selectUTxOEntries :: UTxO -> Int -> Gen ([(TxIn, TxOut)], UTxO)
+selectUTxOEntries = (coerce .) . selectMapEntries . unUTxO
```

Comment:
Is `coerce` the same as `fromIntegral` here?

Reply:
> Is `coerce` the same as `fromIntegral` here?

Here `coerce` has the following type:
```hs
coerce
    :: Gen ([(TxIn, TxOut)], Map TxIn TxOut)
    -> Gen ([(TxIn, TxOut)], UTxO          )
```

It's equivalent (in this case) to `fmap (fmap UTxO)`:
```hs
selectUTxOEntries = (fmap (fmap UTxO) .) . selectMapEntries . unUTxO
```

Reply:
Hi @sevanspowell 

I've changed this use of `coerce` to `fmap (fmap UTxO)` in cbbb455b20d487641751d84e8add4bca343fe5b4.